### PR TITLE
Refactor Help Menu UI

### LINKED ISSUE #51: Refactor Help Menu UI
**Is your feature request related to a problem? Please describe.**
Help menu currently only contains a URL link to the user guide. This workflow of opening up the help menu just to get the URL link to paste into a browser to view the user guide is rather cumbersome.

**Describe the solution you'd like**
We can consider:
- Adding help messages for each commands directly into the help menu, so users don't have to navigate to the user guide. This would mean that we have to maintain both sets of help messages moving forward.
- If we're just redirecting the users to the user guide, maybe the app can just automatically open the user guide in the browser when the help menu is triggered.

### PR DESCRIPTION
**Pull request by celeschai**
Closes #51

### CODE REVIEW

File: src/main/java/careconnect/ui/MainWindow.java [hunk 1]
```diff
@@ -153,14 +155,15 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
...
      */
     @FXML
-    public void handleHelp() {
-        if (!helpWindow.isShowing()) {
-            helpWindow.show();
-        } else {
-            helpWindow.focus();
+    public void handleUgLink() {
+        try {
+            URI link = new URI("https://ay2425s1-cs2103t-w13-2.github.io/tp/UserGuide.html#quick-start");
```
Comment (mongj) [inline src/main/java/careconnect/ui/MainWindow.java:163]: It'd be good if this can be abstracted out into a separate variable :)

**Review by mongj — CHANGES_REQUESTED** [review]

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -29,7 +29,9 @@
             <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
           </Menu>
           <Menu mnemonicParsing="false" text="Help">
-            <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
+            <MenuItem fx:id="helpMenuItem" styleClass="link" text="CareConnect User Guide on Web"
```
Comment (AppleJem) [inline src/main/resources/view/MainWindow.fxml]: There seem to be a UI issue where the help window shows some extra text
image

**Review by mongj — APPROVED** [review]
Thanks for the change! LGTM

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -29,7 +29,9 @@
             <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
           </Menu>
           <Menu mnemonicParsing="false" text="Help">
-            <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
```
Comment (AppleJem) [inline src/main/resources/view/MainWindow.fxml:32]: is the fx:id supposed to be unique?

**Review by AppleJem — COMMENTED** [review]
Super cool feature! There are some smol things that might need changing?

File: src/main/resources/view/MainWindow.fxml (hunk 2 above)
Comment (AppleJem) [reply src/main/resources/view/MainWindow.fxml]: Ah I see it's a keyboard shortcut feature

**Pull request merged**